lib/elixir_ic/search/upload.ex

```diff
@@ -0,0 +1,29 @@
+defmodule ElixirIc.Search.Upload do
+  alias Ecto.Multi
+  alias ElixirIc.Accounts.User
+  alias ElixirIc.Repo
+  alias ElixirIc.Search.Keyword
+
+  @max_keywords 100
+
+  def process(%User{} = user, %Plug.Upload{} = upload) do
+    upload.path
+    |> keyword_values()
+    |> Enum.reduce(Multi.new(), fn name, multi ->
+      Multi.insert(
+        multi,
+        {:keyword, name},
```

Comment:
Not sure, if this one will raise an error when CSV contains the duplicate keyword name. Because Multi name can't have a duplicate name.

Reply:
you're right, if file has duplicates it will throw error. looking into it now